feature-configs/cn-ran-overlays/README.md

```diff
@@ -0,0 +1,25 @@
+# cn-ran-overlays
+
+This folder contains an example configurations for the RAN sites.
+Current example is focused on DU. CU profile can be built by
+- Replacing PTP by NTP (Work in progress)
+- Using stock kernel instead of RT kernel
+- Skipping FEC (Work in progress)
+
+
+## Overview
+
+The [`all-in-one`](all-in-one) directory contains the Kustomize profile for deployment of DU integration features, namely:
```

Comment:
we might wanna call it du-profile instead of all-in-one ?

Reply:
I would rather call it "ran-profile" and then in the README explain that CU is done by this and that modification. Agree?